### ISSUE TITLE
GDPR Research

### ISSUE THREAD

**Issue by patrickdougall** [SCPR/SCPRv4#1187]
Timeboxed

AC:
- [ ] Research GDPR requirements, and how SCPR may or may not be liable
- [ ] Determine process (if any) for removing PII from our existing systems

**Comment by patrickdougall** [comment]
tl;dr we don't appear to be obligated under GDPR to do much of anything. Our PII collection happens through email signups (which already use double opt-in and a preference center), and membership transactions. NPRDS administers our Google Analytics account, which should absolve us of responsibility around our web data. Given the very small amount of personal data we have for EU member state citizens, we don't appear to be in any danger, and it appears that most other NPR stations are in a similar situation.

A key piece of GDPR is data expiry provisions - we are currently set (apparently by NPR) to never automatically expire our data. We may want to revisit this in the future. Our recent redirection to APMG's privacy policy should also help us.
